### PR TITLE
Move library from github enterprise to zalando-incubator

### PR DESCRIPTION
**Pull request by qlibin**
This is a draft of a library aimed to implement Tarbela event producer API.

Tarbela project: https://github.com/zalando-incubator/tarbela

### CODE REVIEW

File: src/main/java/de/zalando/wholesale/tarbelaproducer/EnableTarbelaProducer.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package de.zalando.wholesale.tarbelaproducer;
```
Comment (ePaul) [inline src/main/java/de/zalando/wholesale/tarbelaproducer/EnableTarbelaProducer.java]: I guess I should have said this before, but [the recommended base package name for our open source stuff is org.zalando, not de.zalando](https://github.com/zalando/zalando-howto-open-source#official-namespace).

**Comment by ePaul** [comment]
Not sure if it would be better to move the whole history of the library (i.e. the changes done internally) too, or if this is enough this way.

File: src/main/java/de/zalando/wholesale/tarbelaproducer/EnableTarbelaProducer.java (hunk 1 above)
Comment (qlibin) [inline src/main/java/de/zalando/wholesale/tarbelaproducer/EnableTarbelaProducer.java]: Sure, thank you for the reminder. I've completely forgot about package naming

**Comment by goekhanak** [comment]
:+1:

**Comment by raphaelbauer** [comment]
👍

**Comment by raphaelbauer** [comment]
get it out into the wild! :D

**Pull request closed without merging**